api/package.json

```
@@ -5,10 +5,9 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "dev": "cross-env NODE_ENV=development nodemon ./src/index.js",
-    "start": "NODE_ENV=production node ./src/index.js",
-    "start:test": "NODE_ENV=test node ./src/index.js",
-    "initdb": "node ./src/db/scriptInit.js"
+    "dev": "cross-env NODE_ENV=development npx sequelize-cli db:migrate && nodemon ./src/index.js",
```

dans `index.js`

```
console.log(`NODE_ENV: ${process.env.NODE_ENV}`); // undefined
```
```suggestion
    "dev": "cross-env NODE_ENV=development npx sequelize-cli db:migrate && cross-env NODE_ENV=development nodemon ./src/index.js",
```

je ne sais pas si c'est la bonne manière de faire mais...